src/System.Reflection.Context/dir.props

```diff
@@ -2,8 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.3.0</AssemblyVersion>
-    <AssemblyKey>ECMA</AssemblyKey>
-    <IsUAP>true</IsUAP>
+    <AssemblyVersion>4.0.4.0</AssemblyVersion>
+    <AssemblyKey>Open</AssemblyKey>
```

Comment:
You cannot change the key. If it exists as ECMA in .NET Framework it should be ECMA here. Also I suspect the assembly version has already been bumped so you don't need to do that.